ocaml/utils/warnings.ml

```diff
@@ -1217,6 +1223,11 @@ let message = function
       Printf.sprintf
         "This [@unboxed] attribute cannot be used.\n\
          The type of this value does not allow unboxing."
+  | Mod_by_top modifier ->
+      Printf.sprintf
+        "The modifier %s is a top.\n\
+         Modifying by a top element is a no-op."
```

Comment:
I don't see this string in the testsuite. Add some tests with this warning enabled.

Reply:
This is the warning where the site that raises it is commented out because `transl_modifier` is called multiple times on the same input, causing the warning to be repeated. I guess I should do the ugly hashtable approach for now since I haven't gotten around to fixing the duplicate calls.

Reply:
Ah, right. No, don't do that. We can just merge this as-is, and then come back and do the right thing. The hashtable approach is not super easy, and it will have its own bugs.